app/models/busstop_node.rb

```diff
@@ -165,6 +165,7 @@ def self.allUnvisitedNode
 	end
 
 	def self.findRoute(startID,endID)
+		allRoutesFound = Array.new
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Use snake_case for variable names.